src/components/kytos/napp/Toolbar.vue

```diff
@@ -8,9 +8,10 @@
   </div>
  </template>
  
- <script>
- import Vue from 'vue'
- import { loadModule } from "vue3-sfc-loader"
+ <script type="module">
+ import { defineAsyncComponent } from 'vue'
+ import { loadModule } from 'https://cdn.jsdelivr.net/npm/vue3-sfc-loader@0.9.5/dist/vue3-sfc-loader.esm.js' //'vue3-sfc-loader'
```

Comment:
I think you can revert to "vue3-sfc-loader". It will be hard to track this version of the library if it is hardcoded.